.github/workflows/post_merge.yml

```diff
@@ -0,0 +1,39 @@
+name: Post merge
+
+on:
+  push:
+    branches:
+      - 1.x
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    name: Deploy documentation
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
+      - run: sed -i 's/${POST_MERGE_RUN_ID}/${{ github.run_id }}/g' documentation/how-to-use-appyx/sample-apps.md
+      - run: pip install mkdocs-material
+      - run: mkdocs gh-deploy --force
+
+  build-sample-app:
+    name: Build sample app
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: '11'
+      - uses: gradle/wrapper-validation-action@v1
+      - uses: gradle/gradle-build-action@v2
```

Comment:
What do you think about `cache-read-only: true` here?
Because changes in both core and samples will trigger cache invalidation, does not really makes sense to keep it.

Reply:
Sure. I can add it